Scrum 2765

ianlongden opened the pull request.

azurebrd commented on agr_literature_service/lit_processing/oneoff_scripts/make_subset_db/make_subset.py:
```diff
@@ -14,24 +14,102 @@
     env ORIG_XXX used for source reference db details.
     If ORIG_XXX is defined use that for source db details.
-    If ORIG_XXX NOT defined then use PSQL_XXX details.
+    If ORIG_XXX NOT defined then PSQL_XXX details wil be used.
```
typo "will be used" but not important

ianlongden replied:
fixed. Thanks

azurebrd commented on agr_literature_service/lit_processing/oneoff_scripts/make_subset_db/make_subset.py:
```diff
@@ -14,24 +14,102 @@
 parser = argparse.ArgumentParser(formatter_class=argparse.RawDescriptionHelpFormatter, description=helptext)
 parser.add_argument('-l', '--limit', help='Limit to x references', type=int, default=2000, required=False)
 parser.add_argument('-v', '--verbose', help="verbosity, minumum=0, headlines=1, full=2,", type=int, default=1, required=False)
+parser.add_argument('-d', '--dump_dir', help='Directory to dump schemas and dbs too', type=str, required=True)
```
probably typo "and dbs to", but maybe not

ianlongden replied:
fixed. Thanks

azurebrd commented on agr_literature_service/lit_processing/oneoff_scripts/make_subset_db/make_subset.py:
```diff
@@ -14,24 +14,102 @@
 parser.add_argument('-l', '--limit', help='Limit to x references', type=int, default=2000, required=False)
 parser.add_argument('-v', '--verbose', help="verbosity, minumum=0, headlines=1, full=2,", type=int, default=1, required=False)
+parser.add_argument('-d', '--dump_dir', help='Directory to dump schemas and dbs too', type=str, required=True)
+parser.add_argument('-f', '--full_orig_dump', help='Dump the original database in full aswell', type=bool, default=False, required=False)
```
even smaller typo "as well"

ianlongden replied:
fixed. Thanks

azurebrd reviewed (dismissed):
Thanks @ianlongden .  Great to see how to alter the triggers to disable / enable, for future large loads / updates.  I'm kinda wondering if the make_subset.py should be in oneoff_scripts/ or somewhere else, since they'll keep getting used, while in theory the oneoff_scripts/ might get purged at some point ?  I'd defer to @valearna and @sweng66 on the organization of the repo.

ianlongden commented:
I have updated this a fair bit since the last commit as I had time this morning. Mainly code repetition removal and added resources for the test references. This may need another quick review.

azurebrd reviewed (approved):
Cool !
